test/remove-empty-paragraphs.test.js

```diff
@@ -0,0 +1,35 @@
+const toDom = require('../src/backend/utils/html/dom');
+const removeEmptyParagraphs = require('../src/backend/utils/html/remove-empty-paragraphs');
+
+describe('Remove no content anchor tags', () => {
+  test('should remove <p></p>', () => {
+    const htmlData = toDom('<div><p></p></div>');
+    removeEmptyParagraphs(htmlData);
+
+    const expectedHtml = '<div></div>';
+    expect(htmlData.window.document.body.innerHTML).toEqual(expectedHtml);
+  });
+
+  test('should remove <p>   </p> (spaces)', () => {
+    const htmlData = toDom('<div><p>   </p></div>');
+    removeEmptyParagraphs(htmlData);
+
+    const expectedHtml = '<div></div>';
+    expect(htmlData.window.document.body.innerHTML).toEqual(expectedHtml);
+  });
+
+  test('should remove <p>&#9&#9</p> (tabs)', () => {
```

Comment:
Can you move the unicode entities into the test data vs. the test comment?  The reason I'm pressing on this is that you can't see the special whitespace characters in these tests, and so it's easy for someone to remove them by accident, or not understand why things are failing later if the test data is literally invisible.  If you use explicit entities vs. literal characters, the test data becomes something that can be read vs. trusted.

Reply:
Oh my, I just realized that I put them in the wrong area. Thank you for pointing that out, I'll fix it ASAP.